packages/twenty-front/src/modules/activities/calendar/components/CalendarCurrentEventCursor.tsx

```diff
@@ -32,7 +32,7 @@ const StyledCurrentEventCursor = styled(motion.div)`
   transform: translateY(-50%);
 
   &::before {
-    background-color: ${({ theme }) => theme.font.color.danger};
+    background-color: ${({ theme }) => theme.color.red};
```

Comment:
this should not be necessary if we keep font.color.danger = color.red, see comment below